app/views/admin/index.html.erb

```diff
@@ -13,7 +13,7 @@
 
 <div class="column">
   <section class="top_customers">
-    <h3>  Top 5 Customers</h3>
+    <h3>  Top 5 Customers </h3>
```

Comment:
Just curious, what's the reason that you're leaving spaces around the heading text?